ui/i18n/messages.json

```diff
@@ -31,6 +31,8 @@
   "command.search": "Search",
   "command.select": "Select",
   "command.sort": "Sort",
+  "command.sort.ascend": "Sort in ascending order",
+  "command.sort.descend": "Sort in descending order",
```

Comment:
Once the code is merged, please do not forget to create an NXDOC ticket for these.

Reply:
https://jira.nuxeo.com/browse/NXDOC-2424